src/etcd_client.rs

```diff
@@ -20,14 +19,9 @@ impl EtcdClient {
             etcd_rs::Client::connect(etcd_rs::ClientConfig {
                 endpoints: etcd_address_vec.clone(),
                 auth: None,
+                tls: None,
             })
             .await
```

Comment:
add `.context(format!("failed to build etcd client to addresses={:?}", etcd_address_vec))`